pkcs8/src/attributes.rs

```diff
@@ -0,0 +1,30 @@
+use core::convert::TryFrom;
+
+use der::{Encodable, Encoder, Tag, Tagged};
+
+#[derive(Debug)]
+pub(crate) struct _AttributesStub;
```

Comment:
Nit: I think the `_` is unnecessary, especially as this is a `pub(crate)` type to begin with.

Maybe just call it `Attributes`?

Reply:
It *is* a stub, but imma do that and then add a `TODO` to it.